content/pt-br/pod.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Pod
+status: Completed
+category: conceito
+tags: ["infraestrutura", "fundamento", ""]
+---
+
+Em um ambiente [Kubernetes](/kubernetes/), um pod atua como a unidade de implantação mais básica.
+Ele representa um bloco essencial para implantar e gerenciar aplicações containerizadas.
+Cada pod contém uma única instância de aplicação e pode conter um ou mais [containers](/container/).
```

Comment:
containers -> contêineres
Please check all the other occurrences.

Reply:
Done ✅